source/solutions-library/IoT-datahub-manufacturing.txt

```diff
@@ -0,0 +1,268 @@
+an IoT landscape, taking data from disparate edge devices and using that
+data to power smart applications for a variety of data consumers.
+
+Other Applicable Industries and Use Cases
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Automotive**: Automotive is a subset of manufacturing. Car makers need
+to develop efficient production factories and manage a complex inventory
+of parts and services in order to maximize throughput of finished cars.
+
+**Telecommunications**: Telecommunications is at the heart of edge and
+IoT as more people around the world conduct their lives and businesses
+through mobile and connected devices.
+
+**Healthcare**: The healthcare industry is proving to be one of the
+largest industries affected by IoT. Wearable and connected devices are
+critical for providing care both within medical institutions and
+increasingly in other areas, such as within a patient’s own home and on
+their personal devices.
+
+Reference Architectures
+-----------------------
+
+**Without MongoDB**:
+
+It’s common in many manufacturing environments to have unique systems
+running their own proprietary software, often with a relational database
+in the background, or a niche database covering time-series and graph
+use cases.
+
+The challenge with this approach is that as the number and type of
+databases becomes unmanageable, architectural complexity increases, and
+it becomes more difficult to move data between disparate systems and
+connect them to newer, smarter applications.
+
+.. figure:: /includes/images/industry-solutions/Azure Smart IoT Hub - Ref Arch Before state.svg
+   :figwidth: 1200px
+   :alt: Reference Architecture Without MongoDB
+
+   Figure 1: Reference Architecture Without MongoDB
+
+**With MongoDB**:
+
+With MongoDB Atlas, architectural complexity is reduced, as the
+versatility of the document model allows teams to structure data in
+any way their application needs. The expressive API and indexing also
+allow teams to query data any way they want.
+
+.. figure:: /includes/images/industry-solutions/azure-smarthub-refarch-afterstate.svg
+   :figwidth: 1200px
+   :alt: Reference Architecture With MongoDB
+
+   Figure 2: Reference Architecture With MongoDB
+
+Data Model Approach
+-------------------
+
+Sensors in a factory may be streaming time-series data for multiple
+machines, capturing the following data points:
+
+- Product Type
+- Air Temperature (K)
+- Process Temperature (K)
+- Rotational Speed (rpm)
+- Torque (Nm)
+- Tool Wear (min)
+
+In a relational database, you would likely have to create multiple
+tables to effectively store this varied data. But with MongoDB and
+the flexible document data model, you can use a single collection to
+store all sensor data coming in using 
+`time-series collections <https://www.mongodb.com/docs/manual/core/timeseries/timeseries-procedures/#insert-measurements-into-a-time-series-collection>`__ .
+For example:
+
+   .. code-block:: sh
```

Comment:
This line should start at column 1, meaning we'll need to adjust the indentation for the contained code accordingly.

Reply:
It shouldn't be 🤔 I see that right now with the current formatting we have this error:

> ERROR(solutions-library/IoT-datahub-manufacturing.txt:119ish): Unexpected indentation

This is because it's expecting `.. code-block:: sh` to appear at column 1, rather than indented as it is currently. When you move it to column 1, what is the error you're seeing?